src/hotspot/share/prims/forte.cpp

```diff
@@ -591,6 +591,9 @@ void AsyncGetCallTrace(ASGCT_CallTrace *trace, jint depth, void* ucontext) {
     return;
   }
 
+  // !important! make sure all return points will reset the IN_ASGCT flag to 'false'
```

Comment:
Please change it to something like "// !important! make sure all to call set_in_asgct(false) before every return" as this is clearer in the context of the forte.cpp and omits mentioning the encapsulated state.

Reply:
👌 
I have modified the comment wording

Reply:
There is an "all" to much in the new version, I forgot to remove it in my suggestion